tsantos/serializer-bundle/1.0/config/packages/tsantos_serializer.yaml

```diff
@@ -0,0 +1,6 @@
+tsantos_serializer:
+    generation_strategy: file_not_exists
+#    mapping:
```

Comment:
Maybe it make more sense to uncomment these?

Reply:
The comments are just templates to indicate how to configure such option.

Reply:
I do not prefer config in comments like this. I suggest the following: 
- Add a link to your bundles documentation
- Remove these comments OR (if it makes sense) un-comment these lines.